README.md

```diff
@@ -1,17 +1,139 @@
-# UCAN IPLD Schema
+# UCAN IPLD Schema v0.1.0
 
 ## Editors
 
-* _
+* [Irakli Gozalishvili](https://github.com/Gozala), DAG House
 
 ## Authors
 
-* _
+* [Hugo Dias](https://github.com/hugomrdias), DAG House
+* [Irakli Gozalishvili](https://github.com/Gozala), DAG House
+* [Brooklyn Zelenka](https://github.com/expede), [Fission](https://fission.codes)
+
+## Language
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC2119](https://datatracker.ietf.org/doc/html/rfc2119).
 
 # 0 Abstract
 
-## Language
+[Interplanetary Linked Data (IPLD)](https://ipld.io/) is a consistent, highly general data model for content addressed linked data forming any DAG. This data model provides a convenient pivot between many serializations of the same information. Being a chained token model, UCANs can be very naturally expressed via IPLD. However, JWTs are not fully deterministic due to whitespace and key ordering. This specification defines an IPLD Schema for UCAN, and a canonical JWT serialization for compatibility with all other clients.
+
+# 1 Motivation
+
+The [core UCAN specification](https://github.com/ucan-wg/spec) is defined as a JWT to make it easily adoptable with existing tools, and as a common format for all implementations. There are many cases where different encodings are preferred for reasons such as compactness, machine-efficient formats, and so on. This specification outlines a format based on IPLD that can be deterministically encoded and decoded between many serialization formats, while still being able to encode as JWT for compatibility.
+
+# 2 IPLD Schema
+
+Unlike a JWT, the IPLD encoding of UCAN does not require separate header, claims, and signature fields.
+
+```ipldsch
+type UCAN struct {
+  ucv String 
```

Comment:
Right now we're using `v` for this field, could we use that instead ?

```suggestion
  v String 
```

Alternatively I've considered using a link to a IPLD schema definition, which might be a bit more explicit that a version string, but it would make comparing CIDs more complicated.

However that would imply no simple way to compare versions, that said I'm not sure that is all that useful.

Reply:
Fine by me. I even had a previous version where every field was one character long, but then it was a bit limiting, and matching them with the JWT fields seemed logical. I'm happy with whatever is easiest for you.